### ISSUE TITLE
Pulsar-8 branch won't build for Xilinx PicoZed - SSL error during bitbake for WebOb

### ISSUE THREAD

**Issue by rcordeau** [WindRiver-OpenSourceLabs/wr-core#85]
Going through the readme. After sourcing the env setup, I just wanted to run the bitbake to get stuff set up. "bitbake cube-server cube-dom0 cube-essential" But, it failed out with:
`
WARNING: Failed to fetch URL http://pypi.python.org/packages/source/W/WebOb/WebOb-1.4.1.tar.gz, attempting MIRRORS if available
ERROR: Fetcher failure: Fetch command failed with exit code 8, output:
http://pypi.python.org/packages/source/W/WebOb/WebOb-1.4.1.tar.gz:
2017-10-27 10:11:34 ERROR 403: SSL is required.
`
`
ERROR: Function failed: Fetcher failure for URL: 'http://pypi.python.org/packages/source/W/WebOb/WebOb-1.4.1.tar.gz'. Unable to fetch URL from any source.
ERROR: Logfile of failure stored in: /home/rcordeau/repos/wr-core/build-xilinx-zynq/tmp/work/armv7a-vfp-neon-wrs-linux-gnueabi/python-webob/1.4.1-r0/temp/log.do_fetch.93852
ERROR: Task 6218 (/home/rcordeau/repos/wr-core/layers/meta-virtualization/recipes-devtools/python/python-webob_1.4.1.bb, do_fetch) failed with exit code '1'
`
So, basically, it needs to be grabbing WebOb from https rather than http when going out to pypi to get the package.  However, I don't know how to specify this anywhere. Anyone have any thoughts?

Thanks,
Rob

**Comment by rcordeau** [comment]
Never mind - I found the bb config for WebOb in the meta-virtualization directory. I'll submit a PR for the https fix.
